### ISSUE TITLE
Using rust bindgen

### ISSUE THREAD

**Issue by h4llow3En** [h4llow3En/mac-notification-sys#24]
Maybe we could use rusts bindgen (https://rust-lang.github.io/rust-bindgen/objc.html) instead of using the objc crates, which may have limited functionality.

@hoodie  Would that be a better opportunity to solve #22?

**Comment by hoodie** [comment]
that would be a major overhaul, but perhaps a neat idea. Any idea how to get started? do you maybe wanna do a pairing session on that?

**Comment by h4llow3En** [comment]
Yeah, it would be a major overhaul. But it was something I read while reading about the unions mentioned in #22.

I have no idea yet, just wanted to write it down before I forget it.

A pairing session would be nice for this major task.
